README.md

```diff
@@ -35,7 +39,16 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+
+// Right off the bat, it looks as thought wishlist_items is an array of items, so a simple findOneAndUpdate is probably not ideal.  Here's my take...
+Instructor.findOne({name: "Andy"}).then((error, result) =>{
+  if(error){
+    throw(error)
+  } else {
+  result.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold"})
+  result.save
+  }
+})
 ```
 
```

Comment:
Your approach is fine, but I don't see the advantage over using findOneAndUpdate. I might use that for the sake of semanticity and conciseness.